src/main/java/io/strimzi/kafka/bridge/converter/MessageConverter.java

```diff
@@ -42,4 +43,11 @@
 	 * @return	message
 	 */
 	M toMessage(String address, KafkaConsumerRecord<K, V> record);
+
+	/**
+	 * Converts multiple KafkaConsumerRecord from KafkaConsumerRecords to a message
+	 * @param records KafkaRecords to convert
+	 * @return message
+	 */
+	M toMessages(KafkaConsumerRecords<K, V> records);
```

Comment:
I don't think that it works. For AMQP for example you cannot have `toMessages` plural and then returning just a single `Message`; it should be at least a collection (i.e. `Collection<M>`). But then it doesn't work well with HTTP where you want a `Buffer` containing all messages and not a `Collection<Buffer>`.
Not sure if it works but you could have `MessageConverter<K, V, M, L>` and having `toMessages` returning `C` where for the AMQP implementatin is a `Collection<M>` and for HTTP `M` and `C` are the same and they are `Buffer`. Wdyt?